ask-sdk-model-runtime/ask_sdk_model_runtime/exceptions.py

```diff
@@ -31,10 +35,15 @@ class ServiceException(Exception):
     :type body: object
     """
 
-    def __init__(self, message, status_code, headers, body):
-        # type: (str, int, Optional[List[Tuple[str, str]]], Optional[T]) -> None
+    def __init__(self, message,  # type: str
+                 status_code,  # type: int
+                 headers,  # type: Optional[List[Tuple[str, str]]]
+                 body  # type: Optional[T]
+                 ):
+        # type: (...) -> None
```

Comment:
Although I like this formatting of mentioning the type hints for each constructor parameter, let's stick to the current format we have. I think the last time I tried this, on some python versions, the IDE didn't recognize the type hints.

Reply:
Sure, will revert it back to old format